features/attached_enable.feature

```diff
@@ -706,6 +706,81 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
            | focal   |
            | jammy   |
 
+    @series.xenial
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached enable livepatch
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        Then stdout matches regexp:
+        """
+        Installing canonical-livepatch snap
+        Canonical livepatch enabled
+        """
+        When I run `pro status` with sudo
+        Then stdout matches regexp:
+        """
+        livepatch +yes +enabled
+        """
+        When I run `pro api u.pro.security.status.reboot.v1` with sudo
+        Then stdout matches regexp:
+        """
+        {"_schema_version": "v1", "data": {"attributes": {"reboot_required": "no"}, "meta": {"environment_vars": \[\]}, "type": "RebootStatus"}, "errors": \[\], "result": "success", "version": ".*", "warnings": \[\]}
+        """
+        When I run `ua reboot-required` as non-root
```

Comment:
I'm a little worried about how this will fit in with the _next_ similar request.

If we implement similar things to needsrestart, then we'll need to have another cli command to restart certain services, for example.

Maybe we should nest these system status/management functions under something else? like `pro system reboot-required` and `pro system restart-services`... Just wondering how we can start integrating thesse things without adding a new toplevel command for everything